.github/actions/build-docker/action.yml

```diff
@@ -74,14 +85,14 @@ runs:
       uses: docker/build-push-action@v3
       with:
         context: .
-        tags: ${{ steps.get-image.outputs.result }}
+        tags: ${{ steps.get-images.outputs.result }}
         load: ${{ inputs.push != 'true' }}
         push: ${{ inputs.push }}
         no-cache: true
         platforms: ${{ inputs.platforms }}
         build-args: |
           DISTBALL=${{ steps.get-distball.outputs.result }}
           DATE=${{ steps.get-date.outputs.result }}
-          REVISION=${{ github.sha }}
-          VERSION=${{ steps.get-version.outputs.result }}
+          REVISION=${{ inputs.revision != '' && inputs.revision || github.sha }}
+          VERSION=${{ inputs.version != '' && inputs.version || steps.get-version.outputs.result }}
```

Comment:
:x: Using `steps.get-version.outputs.result` as a fallback seems wrong as that might be something like `camunda/zeebe:8.2.0, camunda/zeebe:latest`

Reply:
seems you may have confused that with the `get-images` step? the output of `get-version` is just the plain version [either from input or determined via mvn command](https://github.com/camunda/zeebe/pull/11240/files/cd04e42c60937a2051023556f2beafced28dfb79#diff-9288d890756492a0e0289f15c477f58821f3a2d9af4274386b09003a4ef57354R67)

Reply:
Oops, yeah you are right :+1: